### PR TITLE
Baseline - Add AWS install mods. Closes #452

### LINKED ISSUE #452: Baseline - Add AWS install mods
- [x]  Add outputs.tf
- [x]  Add variables.tf
- [x]  Add providers.tf
- [ ]  Add smart_folder.tf
- [x]  Create README.md
- [ ]  Categories polices
- [ ]  Add guardrail (category) links
- [x]  Add mod links
- [ ]  Refactor to be in sync with baselines

### PR DESCRIPTION
**Pull request by krishna5891**
Closes #452

### CODE REVIEW

File: baselines/getting_started/aws/aws_mods/demo.tfvars [hunk 1]
```diff
@@ -0,0 +1,124 @@
+variable "mod_list" {
```
Comment (omerosaienni) [inline baselines/getting_started/aws/aws_mods/demo.tfvars:1]: This should be in variables.tf file, an example of the values should be in the demo.tfvars file. 

These default values can then be listed without comment. 

See:
- [example variables](https://github.com/turbot/tdk/blob/master/baselines/getting_started/aws/aws_check_regions/variables.tf)
- [example demo](https://github.com/turbot/tdk/blob/master/baselines/getting_started/aws/aws_check_regions/demo.tfvars)

File: baselines/getting_started/aws/aws_mods/variables.tf [hunk 2]
```diff
@@ -0,0 +1,129 @@
+variable "mod_list" {
```
Comment (omerosaienni) [inline baselines/getting_started/aws/aws_mods/variables.tf:1]: This definition is in both the variables and the demo file? 
How does this work? 

Have you tried running the script against an environment to check that it works?

**Review by omerosaienni — CHANGES_REQUESTED** [review]

File: baselines/getting_started/aws/aws_mods/demo.tfvars (hunk 1 above)
Comment (krishna5891) [reply baselines/getting_started/aws/aws_mods/demo.tfvars]: Updated the changes as per the examples.

File: baselines/getting_started/aws/aws_mods/variables.tf (hunk 2 above)
Comment (krishna5891) [reply baselines/getting_started/aws/aws_mods/variables.tf:1]: I have changed the definition in demo and variables as below,

demo.tfvars has the example values.
variables.tf has the full mod list.

**Review by omerosaienni — APPROVED** [review]

**Pull request merged**